### PR TITLE
Add zone detail states

### PR DESCRIPTION
**Pull request by Kongkille**
## Issue
https://linear.app/electricitymaps/issue/ELE-1513/zone-detail-data-unavailable-states
https://linear.app/electricitymaps/issue/ELE-1499/[ui]-implement-chart-overlay-for-unavailable-data

## Description

Ensures zonedetail looks reasonable during other states

- Loading
- No information
- Error
- No real-time data

There's some missing styling on the panel overall, but since we're still working on some of the charts I'm waiting a bit

### Preview

https://user-images.githubusercontent.com/45588799/207855587-55041fa6-6b53-4dbd-bace-7141b8811a64.mov

### CODE REVIEW

File: web/src/features/charts/bar-breakdown/EmptyBarBreakdownChart.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+  // eslint-disable-next-line unicorn/consistent-function-scoping
+  const formatTick = (t: number) => {
+    // TODO: format tick depending on displayByEmissions
+    return `${t} GW`;
+  };
+
+  return (
+    <>
+      {' '}
```
Comment (tonypls) [inline web/src/features/charts/bar-breakdown/EmptyBarBreakdownChart.tsx:58]: Is this required?

File: web/src/features/charts/bar-breakdown/EmptyBarBreakdownChart.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          {productionData.map((d, index) => (
+            <Row
+              key={d.mode}
+              index={index}
+              label={__(d.mode)}
+              width={width}
+              scale={co2Scale}
+              value={Math.abs(d.tCo2eqPerMin)}
+              isMobile={Boolean(isMobile)}
+              onMouseOut={() => {}}
```
Comment (tonypls) [inline web/src/features/charts/bar-breakdown/EmptyBarBreakdownChart.tsx:83]: Could remove if these are not required?

**Review by tonypls — COMMENTED** [review]
Looks good, would be good to decide on the padding

File: web/src/features/charts/bar-breakdown/EmptyBarBreakdownChart.tsx (hunk 1 above)
Comment (Kongkille) [reply web/src/features/charts/bar-breakdown/EmptyBarBreakdownChart.tsx:58]: looks like something added by the formatter at one point, removed it

File: web/src/features/charts/bar-breakdown/EmptyBarBreakdownChart.tsx (hunk 2 above)
Comment (Kongkille) [reply web/src/features/charts/bar-breakdown/EmptyBarBreakdownChart.tsx:83]: sure, i've made them optional in the row component

**Comment by Kongkille** [comment]
I'm merging this because I want to finalize the zonedetails panel compeltely and I'd like to get #81 in as well before branching off.

**Pull request merged**